### ISSUE TITLE
Message codes for server errors are wrong

### ISSUE THREAD

**Issue by matteobettini** [martindisch/coap-lite#11]
The message codes for server error are not the right ones
```
MessageClass::Response(ResponseType::InternalServerError) => 0x90,
MessageClass::Response(ResponseType::NotImplemented) => 0x91,
MessageClass::Response(ResponseType::BadGateway) => 0x92,
MessageClass::Response(ResponseType::ServiceUnavailable) => 0x93,
MessageClass::Response(ResponseType::GatewayTimeout) => 0x94,
MessageClass::Response(ResponseType::ProxyingNotSupported) => 0x95,
```
while they should be:
```
MessageClass::Response(ResponseType::InternalServerError) => 0xA0,
MessageClass::Response(ResponseType::NotImplemented) => 0xA1,
MessageClass::Response(ResponseType::BadGateway) => 0xA2,
MessageClass::Response(ResponseType::ServiceUnavailable) => 0xA3,
MessageClass::Response(ResponseType::GatewayTimeout) => 0xA4,
MessageClass::Response(ResponseType::ProxyingNotSupported) => 0xA5,
```

**Comment by martindisch** [comment]
Interesting, thanks for catching that (and fixing it right away). I'm currently on holiday and for once did not bring my laptop. I hope it's okay for you if I take a look at and merge it on Monday.

**Comment by matteobettini** [comment]
Thank you very much for the quick correction!

**Comment by martindisch** [comment]
I ran a check on all the other codes as well to be sure, luckily they were all correct already. Merged and published as 0.5.0. Thanks and congrats for your first contribution on GitHub!
